traces/reader.go

```diff
@@ -79,6 +79,9 @@ func (t *TraceLine) Get(title string) string {
 		return fmt.Sprintf("%v (0x%x)", op.Pc/ChunkSize, op.Pc/ChunkSize)
 	case "pc":
 		return fmt.Sprintf("%v (0x%x)", op.Pc, op.Pc)
+	// Depends on Geth EOF support
```

Comment:
Does it really? this can be enabled, no? (and same below)

Reply:
The type comes from Geth.  The ones that remain after my next push are all because the types need to be in Geth.